addons/stock/models/stock_picking.py

```diff
@@ -1352,18 +1352,19 @@ def _attach_sign(self):
         )
         return True
 
-    def _get_aggregated_product_quantities(self):
+    def _get_aggregated_product_quantities(self, move_lines):
```

Comment:
Would be cleaner if the function was on stock.move.line and self was use instead of an additional variable.

Reply:
So smart. Function has been moved to stock.move.line